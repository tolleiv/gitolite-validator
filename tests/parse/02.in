@groupname = hans.maulwurf homer.simpson
