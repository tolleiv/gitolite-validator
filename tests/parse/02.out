root:
  group:
    name: groupname
    members:
      hans.maulwurf
      homer.simpson
